### ISSUE TITLE
typo in composer.json?

### ISSUE THREAD

**Issue by ITwrx** [amsgames/laravel-shop#17]
i get errors during update having to do with the src/commands folder not being accessible. I'm guessing that's because it's src/Commands locally but src/commands in laravel-shop's composer.json.

thanks

**Comment by amostajo** [comment]
Updated composer. Thanks for the finding.

**Comment by ITwrx** [comment]
it seems there's something else too. I couldn't get composer update to run without the below exception so i manually removed laravel-shop and reinstalled via composer but i still get the below exception.

```
[RuntimeException]                                                                                                                              
  Could not scan for classes inside "/srv/http/my_app/vendor/amsgames/laravel-shop/src/commands" which does not appear to be a file nor a folder
```

thanks

**Comment by kriang69** [comment]
![image](https://cloud.githubusercontent.com/assets/7994203/11139949/e5be4ee4-8a07-11e5-8749-247330d25ef3.png)

I renew install via composer require amsgames/laravel-shop. It's show this error. How to solve it?

**Comment by amostajo** [comment]
Run
`composer update`
Fix should be on v0.2.11

**Comment by ITwrx** [comment]
no errors after updating to v0.2.11.
Thanks again.
